content/source/docs/enterprise-beta/api/team-access.html.md

```diff
@@ -102,52 +173,7 @@ $ curl \
   --header "Authorization: Bearer $ATLAS_TOKEN" \
   --header "Content-Type: application/vnd.api+json" \
   --request GET \
-  https://atlas.hashicorp.com/api/v2/team_workspaces/257525
-```
-
-### Sample Response
-
-```json
-{
-  "data": {
-    "type": "team-workspaces",
-    "id": "1",
-    "attributes": { "permission": "read" }
-  }
-}
-```
-
-## Update Team Access to a Workspace
```

Comment:
Is the proper way to do this by removing, then adding?

Reply:
I used the UI as a guide for this, in the UI you remove then add.